draft-ietf-quic-transport.md

```diff
@@ -2692,10 +2709,19 @@ implementation decision, and an implementation should be careful to delay
 conservatively, since any delay is likely to increase application-visible
 latency.
 
-Regular QUIC packets are "containers" of frames; a packet is never retransmitted
-whole.  How an endpoint handles the loss of the frame depends on the type of the
-frame.  Some frames are simply retransmitted, some have their contents moved to
-new frames, and others are never retransmitted.
+Regular QUIC packets are "containers" of frames.  When an endpoint receives an
+ACK frame for one or more transmitted packets, all frames in the acknowledged
+packets are considered to have been delivered to the peer, with one exception.
+A PATH_CHALLENGE frame (see {{frame-path-challenge}}) is used to validate a
```

Comment:
I would prefer to leave this text out because it's more confusing than helpful.  It's ok to consider it delivered to the peer when it's acknowledged, the key is that one must receive a PATH_RESPONSE before the path has been verified, which you say above.

Reply:
It depends on what the semantics of delivery are. What I say above is that an ACK implies that the frame has been received and processed by the peer. I don't think that's a safe assumption with PATH_CHALLENGE, since the entire point of it is to get something besides an ACK.